catalog/kubedb/gen-dbaas-license/main.go

```diff
@@ -27,56 +27,63 @@ import (
 
 func main() {
 	r := configdata.LicenseRestrictions{
-		dbv1a2.ResourceKindDruid: configdata.Restriction{
+		dbv1a2.ResourceKindDruid: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindElasticsearch: configdata.Restriction{
+		}},
+		dbv1.ResourceKindElasticsearch: configdata.Restrictions{{
 			VersionConstraint: "*",
 			Distributions:     []string{"OpenSearch"},
-		},
-		dbv1a2.ResourceKindFerretDB: configdata.Restriction{
+		}},
+		dbv1a2.ResourceKindFerretDB: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindKafka: configdata.Restriction{
+		}},
+		dbv1.ResourceKindKafka: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindMariaDB: configdata.Restriction{
+		}},
+		dbv1.ResourceKindMariaDB: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindMemcached: configdata.Restriction{
+		}},
+		dbv1.ResourceKindMemcached: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindMySQL: configdata.Restriction{
+		}},
+		dbv1.ResourceKindMySQL: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindPerconaXtraDB: configdata.Restriction{
+		}},
+		dbv1.ResourceKindPerconaXtraDB: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindPgBouncer: configdata.Restriction{
+		}},
+		dbv1.ResourceKindPgBouncer: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1a2.ResourceKindPgpool: configdata.Restriction{
+		}},
+		dbv1a2.ResourceKindPgpool: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindPostgres: configdata.Restriction{
+		}},
+		dbv1.ResourceKindPostgres: configdata.Restrictions{{
 			VersionConstraint: "*",
 			Distributions:     []string{"Official"},
-		},
-		dbv1.ResourceKindProxySQL: configdata.Restriction{
+		}},
+		dbv1.ResourceKindProxySQL: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1a2.ResourceKindRabbitmq: configdata.Restriction{
+		}},
+		dbv1a2.ResourceKindRabbitmq: configdata.Restrictions{{
 			VersionConstraint: "*",
-		},
-		dbv1.ResourceKindRedis: configdata.Restriction{
-			VersionConstraint: "<= 7.40",
-		},
-		dbv1a2.ResourceKindSolr: configdata.Restriction{
+		}},
+		dbv1.ResourceKindRedis: configdata.Restrictions{
+			{
+				VersionConstraint: "<= 7.4.0",
+				Distributions:     []string{"Official"},
```

Comment:
https://github.com/kubedb/apimachinery/blob/master/apis/catalog/v1alpha1/redis_version_types.go#L119
Should we change `Redis` -> `Official` in apimachinery ?

Reply:
I guess so.